### DISCUSSION TITLE
2.47 New Keyfile Support

### DISCUSSION THREAD

**Discussion by JustinGrote** [AdminPanteam/pt.KeePassLibStd#8] [Ideas]
Hello!

I use your library in my SecretManagement.KeePass powershell module. I've had a request for the new keyfile support, can you bump this to a merge from 2.47? Thanks!

**Comment by JustinGrote** [comment]
@ArtjomP any interest?

**Reply by ArtjomP** [reply]
Sure!
I'll do it.
I'm a little bit busy at this particular moment. You can make a PR if you have some time.

**Reply by ArtjomP** [reply]
A new version is available on [nuget](https://www.nuget.org/packages/pt.KeePassLibStd/2.47.0.4053-Beta)

**Reply by JustinGrote** [reply]
@ArtjomP Thanks, I got it and it works for .NET core, but .NET framework (Windows Powershell) it won't load the assembly:

![image](https://user-images.githubusercontent.com/15258962/108907825-77027400-75d7-11eb-88b6-441d5627e719.png)

EDIT: Disregard, I didn't unblock the file after downloading the zip. Oops.
